server/ec2spotmanager/models.py

```diff
@@ -97,29 +137,32 @@ def __init__(self, *args, **kwargs):
         # automatically here. You need to explicitly call the
         # deserializeFields method if you need this data.
 
-        super(PoolConfiguration, self).__init__(*args, **kwargs)
 
     def flatten(self):
```

Comment:
Why is there a flatten in the parent, and another flatten here? Shouldn't flatten be provider-independent except for the list of non-scalar fields (dicts, lists)?